application/src/main/java/org/togetherjava/tjbot/commands/free/FreeCommand.java

```diff
@@ -197,12 +198,27 @@ private boolean handleShouldBeProcessed(@NotNull final SlashCommandEvent event)
      *
      * @param channel the text channel the status message will be posted in.
      */
+
+    /*
+     * event.replyEmbeds(new EmbedBuilder() .setDescription(tagSystem.getTag(id).orElseThrow()
+     * .setFooter(event.getUser() + " - used /tag foo") .setColor(TagSystem.AMBIENT_COLOR)
+     * ).queue();
+     */
     public void displayStatus(@NotNull TextChannel channel) {
         final Guild guild = channel.getGuild();
 
+        /*
+         * MessageUtils.generateEmbed(STATUS_TITLE, messageTxt, channel.getJDA().getSelfUser(),
+         * MESSAGE_HIGHLIGHT_COLOR, null)
+         */
+
         String messageTxt = buildStatusMessage(guild);
-        MessageEmbed embed = MessageUtils.generateEmbed(STATUS_TITLE, messageTxt,
-                channel.getJDA().getSelfUser(), MESSAGE_HIGHLIGHT_COLOR, null);
+        MessageEmbed embed = new EmbedBuilder().setTitle(STATUS_TITLE)
+            .setDescription(messageTxt)
+            .setAuthor(String.valueOf(channel.getJDA().getSelfUser()))
```

Comment:
Why are you using `setAuthor` now? Thats wrong. Use `setFooter` like it was in the old code. Dont change behavior. Also, `String.valueOf(...)` is definitely wrong.

Reply:
Now you removed it completely. You have to use `setFooter` in the way it worked before you deleted the method. The username was previously displayed in the footer and you have to replicate that behavior now, otherwise you are changing the behavior.